src/modality-specific-files/task-events.md

```diff
@@ -82,9 +82,10 @@ A guide for using macros can be found at
 Example of the content of the TSV file:
 
 ```Text
-onset	duration	trial_type	response_time	stim_file
-1.23	0.65	start	1.435	images/red_square.jpg
-5.65	0.65	stop	1.739	images/blue_square.jpg
+onset	duration	trial_type	response_time	stim_file	channel	artifact
+1.23	0.65	start	1.435	images/red_square.jpg	n/a	n/a
+5.65	0.65	stop	1.739	images/blue_square.jpg	n/a	n/a
+12.1	2.35	n/a	n/a	n/a	F,1|F,2|Cz	sweat
```

Comment:
The "sweat"  was "musc" in a later example and the "artifact" column was called "annot".  I think the same example should be continued along.